components/AuthWrapper/SignInButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { Button } from '~honeycomb';
+import { useSession, signIn, signOut } from 'next-auth/react';
+
+const SignInButton: FC = () => {
```

Comment:
```suggestion
const SignInButton = (): JSX.Element => {
```

`FC` is pretty useless these days, it doesn't type your `children` or anything anymore and is basically a shortcut to what I have in the suggestion, so I think its best to just be explicit and not rely on a magical shortcut that can change its definition at any time.